### PR TITLE
docs: add alvarocamillont to GDE resources

### PR DESCRIPTION
**Pull request by alvarocamillont**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [X] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -829,5 +829,13 @@
     "website": "https://thekiba.io",
     "bio": "Andrew is a software engineer using Angular and .NET. He spends most of his spare time staying up-to-date, helping other people, and experimenting with web tech.",
     "groups": ["GDE"]
+  },
+  "alvarocamillont": {
+    "name": "Alvaro Camillo Neto",
+    "picture": "alvarocamillont.png",
+    "twitter": "alvarocamillont",
+    "website": "https://alvarocamillont.dev/",
+    "bio": "Alvaro is a software engineer, speaker and instructor in Brazil. He works in the technology industry for over 10 years dedicated to the development of business solutions in a large company in the industry.",
```
Comment (kapunahelewong) [inline aio/content/marketing/contributors.json:838]:
```suggestion
    "bio": "Alvaro is a software engineer, speaker, and instructor in Brazil. He has worked in the technology industry for over 10 years and is dedicated to the development of enterprise business solutions.",
```

What about this?  The first three suggestions are just grammar. The last one, "enterprise", implies "a large company in the industry".

**Review by kapunahelewong — COMMENTED** [review]
Congratulations, @alvarocamillont!! I've just made some small optional suggestions. We will still need marketing approval. Well done!

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
Oops! I forgot to ask if you could get your pic down to less than 30kb. 🙏🏼

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (alvarocamillont) [reply aio/content/marketing/contributors.json:838]: Perfect!

**Comment by alvarocamillont** [comment]
> Oops! I forgot to ask if you could get your pic down to less than 30kb. 🙏🏼

No problem! I sent a new photo!

**Review by kapunahelewong — APPROVED** [review]
LGTM! Thank you, @alvarocamillont!! We just need marketing approval. 🎉

Reviewed-for: global-docs-approvers

**Comment by kapunahelewong** [comment]
To satisfy pull approve 🤞🏼

Reviewed-for: required-minimum-review

**Review by mgechev — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
Hi @alvarocamillont, this PR was approved, but it has a conflict with master branch. Could you please rebase and resolve conflicts, so that we can proceed with merging it? Thank you.

**Comment by alvarocamillont** [comment]
> 
> 
> Hi @alvarocamillont, this PR was approved, but it has a conflict with master branch. Could you please rebase and resolve conflicts, so that we can proceed with merging it? Thank you.

Got it!

**Pull request closed without merging**